src/services/blogs.js

```diff
@@ -0,0 +1,6 @@
+import { checkError, client } from './client.js';
```

Comment:
good work keeping this in it's own file + much more easily accessed for future programming